patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -1,6 +1,13 @@
 --- ../src-base/minecraft/net/minecraft/entity/player/EntityPlayer.java
 +++ ../src-work/minecraft/net/minecraft/entity/player/EntityPlayer.java
-@@ -96,6 +96,11 @@
+@@ -90,12 +90,18 @@
+ import net.minecraft.world.LockCode;
+ import net.minecraft.world.World;
+ import net.minecraft.world.WorldServer;
++import net.minecraftforge.common.ForgeHooks;
```

Comment:
Do not use imports in patches, instead use the fully qualified name where it is used.

Reply:
Fixed in latest commit.